lib/elixir_boilerplate_graphql/elixir_boilerplate_graphql.ex

```diff
@@ -19,6 +19,11 @@ defmodule ElixirBoilerplateGraphQL do
 
     config
     |> Absinthe.Plug.default_pipeline(options)
+    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.IntrospectionCheck, options})
+    |> Pipeline.insert_after(Absinthe.Phase.Document.Result, {AbsintheSecurity.Phase.FieldSuggestionsCheck, options})
```

Comment:
Since `Absinthe.Phase.Document.Result` is the last phase in the default pipeline, we could organize our checks to follow the same order:

https://github.com/absinthe-graphql/absinthe/blob/5b5f24f919bd42e4f6ff9fb8aa90436508a3f8b9/lib/absinthe/pipeline.ex#L52-L123

Reply:
Can you suggest a new order? 🙂 🙏